core/models/program.py

```diff
@@ -1,5 +1,5 @@
 from django.db import models
-
+import re
```

Comment:
Why import re?

Reply:
i was going to add a progam slug then decided against it. will remove